procedures/mobile_automation_setup.rst

```diff
@@ -32,100 +26,62 @@ administrating them in the live Taskcluster website.
-    2. Make sure that all scopes are provided to fulfill requirements in
-       ``.taskcluster.yml`` (when ``is_mozilla_mobile_repo`` is
-       ``true``)
+3.  Add ``.taskcluster.yml`` in the root of the repository. This file
+    tells Taskcluster what to do upon github events happening (push,
+    pr, release, etc). Since we're going to want to run ``taskgraph``
+    to decide what tasks to run, we can take a ``.taskcluster.yml`` from
+    a similarly-configured repository, like `fenix
+    example <https://github.com/mozilla-mobile/fenix/blob/master/.taskcluster.yml>`__)
+
+    -  Update repository and treeherder references to refer to your project,
+       rather than ``fenix``.
+
+4.  Implement ``taskgraph`` configuration for the repository. See the
+    `Fenix configuration <https://github.com/mozilla-mobile/fenix/tree/master/taskcluster>`__.
+    You'll need to implement the following parts:
+
+    -  Define tasks in YAML in ``taskcluster/ci/``
+    -  Define transforms in ``taskcluster/$project_taskgraph/transforms/`` which operate
+       on the tasks defined in the YAML
+    -  Define any custom loaders in ``taskcluster/$project_taskgraph/loader/`` (this is
+       useful in cases like needing to generate a dynamic number of tasks based on an
+       external source, like ``gradle`` or a ``.buildconfig.yml`` file)
+    -  Define ``Dockerfiles`` in ``taskcluster/docker/``
+
+5.  If you have schedule-based automation, create cron hook
+    (copy from the `fenix hook <https://tools.taskcluster.net/hooks/project-releng/cron-task-mozilla-mobile-fenix>`__)
+
+    -  You'll need to update the references from ``fenix``
+    -  Additionally, define a ``.cron.yml``, like `fenix <https://github.com/mozilla-mobile/fenix/blob/master/.cron.yml>`__
+    -  Create some hooks to force-trigger some cron tasks for testing. For example, `fenix has one for forcing nightlies
+       <https://tools.taskcluster.net/hooks/project-releng/cron-task-mozilla-mobile-fenix%2Fnightly>`__
+
+        Note that the hook URL has a ``%2F`` instead of a slash - this is due to a bug with the Taskcluster tools page loading the wrong hook
+
+
+6.  Create and update permissions in ``ci-configuration``.
+
+    1. Install ``ci-admin`` if you haven't already
+
+        1. ``hg clone https://hg.mozilla.org/ci/ci-admin/``
+        2. Set up a ``virtualenv`` and install dependencies
+
+    1. ``hg clone https://hg.mozilla.org/ci/ci-configuration/``
+    2. Update ``projects.yml`` and ``grants.yml`` to add permissions for ``$project``
+    3. Submit your patch for review with `moz-phab <https://github.com/mozilla-conduit/review>`__
+    4. Once it's landed, update to the new revision and apply it
+
+        1. ``ci-admin diff --environment=production``
+        2. If there's no surprises in the diff, apply it: ``ci-admin apply --environment=production``
+        3. If the diff has unexpected contents, you can adjust the ``diff`` and ``apply`` operations with the ``--grep`` flag:
```

Comment:
Nit: Let's explain what's unexpected in this context, and why it's okay to see that. For example: mozilla-{central,beta,release} hooks are regularly updated by ci-admin and they may show up in the diff until someones runs it.

Reply:
Sure, I defined "unexpected" as changes in the diff that are unrelated to the hooks/permissions that were created earlier in the workflow :)